python/semantic_kernel/prompt_template/const.py

```diff
@@ -1,16 +1,9 @@
 # Copyright (c) Microsoft. All rights reserved.
 
-from typing import Literal, get_args
+from typing import Literal
 
-KERNEL_TEMPLATE_FORMAT_NAME_TYPE = Literal["semantic-kernel"]
-KERNEL_TEMPLATE_FORMAT_NAME: KERNEL_TEMPLATE_FORMAT_NAME_TYPE = get_args(KERNEL_TEMPLATE_FORMAT_NAME_TYPE)[0]
-HANDLEBARS_TEMPLATE_FORMAT_NAME_TYPE = Literal["handlebars"]
-HANDLEBARS_TEMPLATE_FORMAT_NAME: HANDLEBARS_TEMPLATE_FORMAT_NAME_TYPE = get_args(HANDLEBARS_TEMPLATE_FORMAT_NAME_TYPE)[
-    0
-]
-JINJA2_TEMPLATE_FORMAT_NAME_TYPE = Literal["jinja2"]
-JINJA2_TEMPLATE_FORMAT_NAME: JINJA2_TEMPLATE_FORMAT_NAME_TYPE = get_args(JINJA2_TEMPLATE_FORMAT_NAME_TYPE)[0]
+KERNEL_TEMPLATE_FORMAT_NAME: Literal["semantic-kernel"] = "semantic-kernel"
```

Comment:
These constants seemed overly complicated for what we were trying to use them for (literals). Pylance also was upset that were trying to use types, and not actual literals, in `kernel_function.py`.